src/main/resources/config.sk

```diff
@@ -31,7 +31,7 @@
 # ==== General Options ====
 
 language: english
-# Which language to use. Currently English, German, Korean, French and Polish are included in the download, but custom languages can be created as well.
+# Which language to use. Currently English, German, Korean, French, Polish and Simplified Chinese are included in the download, but custom languages can be created as well.
```

Comment:
```suggestion
# Which language to use. Currently English, German, Korean, French, Polish and Simplified Chinese (Chinese as the name) are included in the download, but custom languages can be created as well.
```
When you set the language it follows <language>.lang so adding these braces allows the user to know exactly what to type as the node.

Reply:
I changed the "chinese.lang" file name to "simplifiedchinese.lang", and added some comments in config.sk.
I think using spaces in "language" might cause some problems.